eng/Versions.props

```diff
@@ -240,7 +240,8 @@
                like - DarcDependenciesChanged.Microsoft_NET_Workload_Emscripten_Current_Manifest-9_0_100_Transport
     -->
     <MicrosoftNETWorkloadEmscriptenCurrentManifest90100TransportVersion>9.0.0-rc.2.24421.3</MicrosoftNETWorkloadEmscriptenCurrentManifest90100TransportVersion>
-    <MicrosoftNETRuntimeEmscriptenVersion>$(MicrosoftNETWorkloadEmscriptenCurrentManifest90100TransportVersion)</MicrosoftNETRuntimeEmscriptenVersion>
+    <MicrosoftNETWorkloadEmscriptenCurrentManifest90100Version>9.0.0-rc.2.24421.3</MicrosoftNETWorkloadEmscriptenCurrentManifest90100Version>
+    <MicrosoftNETRuntimeEmscriptenVersion>$(MicrosoftNETWorkloadEmscriptenCurrentManifest90100Version)</MicrosoftNETRuntimeEmscriptenVersion>
```

Comment:
This is because there are references to the packages via stable versions?

Reply:
TransportVersion is coming from non-shipping dependency and that did not work when used for references with version defined by MicrosoftNETRuntimeEmscriptenVersion.  I have added another dependency with shipping versioning (MicrosoftNETWorkloadEmscriptenCurrentManifest90100Version) and use that version as for further references of Emscripten.

Reply:
Was the source build version not sufficient?